### DISCUSSION TITLE
Can i trust the affiliate url?

### DISCUSSION THREAD

**Discussion by mandamattosg** [sergioteula/python-amazon-paapi#131] [Help] [answered]
Hello! I just wanted to know if the affiliate url you generate is valid. I'm asking because when i get my affiliate url link from the amazon page, it's a little different from the one you generate with the code.

**Comment by sergioteula** [comment, marked as answer]
The url comes directly from Amazon, so it should be correct. In fact, the only needed thing to be a valid affiliate url is the `tag`.

**Reply by mandamattosg** [reply]
Ok, thank you!!
